lib/modules/datasource/docker/index.ts

```diff
@@ -464,6 +465,41 @@ export class DockerDatasource extends Datasource {
     }
   }
 
+  @cache({
+    namespace: 'datasource-docker-imageconfig',
+    key: (
+      registryHost: string,
+      dockerRepository: string,
+      configDigest: string
+    ) => `${registryHost}:${dockerRepository}@${configDigest}`,
+    ttlMinutes: 1440 * 28,
+  })
+  public async getImageConfig(
+    registryHost: string,
+    dockerRepository: string,
+    configDigest: string
+  ): Promise<HttpResponse<ImageConfig> | undefined> {
+    logger.debug(
```

Comment:
@rarkins should we move this to trace?

Reply:
Why? `getManifestResponse()`, `getLabels()` and `getDigest()` use `logger.debug` for the same purpose. Changing this to `trace` would be inconsistent.

https://github.com/renovatebot/renovate/blob/930efb5896eba1cfa032ca7872ef6cb7c25b00ab/lib/modules/datasource/docker/index.ts#L392-L394

https://github.com/renovatebot/renovate/blob/930efb5896eba1cfa032ca7872ef6cb7c25b00ab/lib/modules/datasource/docker/index.ts#L676

https://github.com/renovatebot/renovate/blob/930efb5896eba1cfa032ca7872ef6cb7c25b00ab/lib/modules/datasource/docker/index.ts#L946-L950

Reply:
We have some inconsistencies across datasources. I think we're probably overlogging for docker.